osu.Game/OsuGameBase.cs

```diff
@@ -528,7 +531,12 @@ public bool Migrate(string path)
 
         protected virtual BatteryInfo CreateBatteryInfo() => null;
 
-        protected virtual Container CreateScalingContainer() => new DrawSizePreservingFillContainer();
+        /// <summary>
+        /// The size at which all game elements would be laid out at, then scaled to the window resolution using the scaling container specified in <see cref="CreateScalingContainer"/>.
+        /// </summary>
+        protected internal virtual Vector2 TargetDrawSize => new Vector2(1024, 768);
+
+        protected virtual Container CreateScalingContainer() => new DrawSizePreservingFillContainer { TargetDrawSize = TargetDrawSize };
```

Comment:
This feels *so* round-about. You're seeing the value here but also reverse setting it using DI on every `Update`? And on iOS the value *is from the scaling container itself*??

I just don't know.